java/ql/src/semmle/code/java/frameworks/android/Intent.qll

```diff
@@ -42,3 +42,12 @@ class IntentGetExtraMethod extends Method, TaintPreservingCallable {
 
   override predicate returnsTaintFrom(int arg) { arg = -1 }
 }
+
+class BundleGetterMethods extends Method, TaintPreservingCallable {
```

Comment:
The qldoc check failed with:
```
Error: The following CodeQL elements are lacking documentation:
semmle/code/java/frameworks/android/Intent.qll  Intent::BundleGetterMethods  class
```
This is part of our effort to ensure that all public classes and predicates in our libraries are documented, so the check disallows the introduction of new undocumented public classes and predicates.
```suggestion
/** A getter on `android.os.BaseBundle` or `android.os.Bundle`. */
class BundleGetterMethod extends Method, TaintPreservingCallable {
```
Also, I just noticed that the class name ought to be singular rather than plural.

Reply:
Thanks @aschackmull for the advice. I've committed required change.